inc/Engine/Optimization/RUCSS/Admin/Subscriber.php

```diff
@@ -38,15 +48,76 @@ public function __construct( Settings $settings, Database $database ) {
 	 */
 	public static function get_subscribed_events() : array {
 		return [
-			'rocket_first_install_options' => [
-				[ 'add_options_first_time' ],
-			],
-			'wp_rocket_upgrade'            => [
+			'rocket_first_install_options'     => 'add_options_first_time',
+			'wp_rocket_upgrade'                => [
 				[ 'set_option_on_update', 13, 2 ],
 			],
+			'switch_theme'                     => 'truncate_used_css',
+			'rocket_rucss_file_changed'        => 'truncate_used_css',
+			'pre_post_update'                  => 'delete_used_css_on_update_or_delete',
```

Comment:
Is this not covered  by `clean_post_cache`?

Reply:
I got the events from WP Rocket. These are the events which clean WP Rocket cache file in case a post is updated, deleted, affected in any way. I am not sure if both are required or if `clean_post_cache` is sufficient. 
Could be there situation when `clean_post_cache` is not called?

Reply:
For the `pre_post_update` one in WP Rocket, it's only for the post status going from published to draft, there is no content change, so it might not be necessary to clear the used CSS?